src/Module/Register.php

```diff
@@ -405,4 +391,31 @@ protected function post(array $request = [])
 
 		return;
 	}
+
+	private function sendNotification(array $user, string $event)
+	{
+		// send email to admins
+		$admins_stmt = DBA::select(
+			'user',
+			['uid', 'language', 'email'],
+			['email' => explode(',', str_replace(' ', '', DI::config()->get('config', 'admin_email')))]
+		);
+
+		// send notification to admins
+		while ($admin = DBA::fetch($admins_stmt)) {
+			DI::notify()->createFromArray([
+				'type'         => Model\Notification\Type::SYSTEM,
+				'event'        => $event,
+				'uid'          => $admin['uid'],
+				'link'         => DI::baseUrl()->get(true) . '/admin/users/',
+				'source_name'  => $user['username'],
+				'source_mail'  => $user['email'],
+				'source_nick'  => $user['nickname'],
+				'source_link'  => DI::baseUrl()->get(true) . '/admin/users/',
```

Comment:
Shouldn't it be the user profile page URL?

Reply:
This is a 1:1 copy of the other notification, so I hope that everything will be fine.

Reply:
Pending users don't have a user profile yet, so I think this one should be slightly different.